test/NuGet.Clients.Tests/NuGet.VisualStudio.Test/IVsPathContextProviderTests.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Moq;
+using Xunit;
+
+namespace NuGet.VisualStudio.Test
+{
+    public class IVsPathContextProviderTests
+    {
+        // Verify interface method can be used across assembly boundaries
+        [Fact]
+        public void TryCreateContext_EmbeddableTest()
+        {
+            var provider = Mock.Of<IVsPathContextProvider>();
```

Comment:
It has always been the style guideline for tests in NuGet, you can take a look around at other tests. We could revisit these.